.github/workflows/automation.yml

```diff
@@ -0,0 +1,26 @@
+name: automation
+
+on:
+  push:
+    branches-ignore:
+      - 'master'
```

Comment:
The pipeline doesn't run on the `master` branch. Following git flow principles, the intention was to let it run on `devel` and routinely merge `devel` into `master`. I guess this should be clarified.

Reply:
IMO a `devel` branch would come with additional merging (where we will not be reviewing anything really, as all ecModels will have already been reviewed) & slower delivery to the community (ecModels might stall in `devel`), with no gains that I can see. @mihai-sysbio @IVANDOMENZAIN what are your thoughts?

Reply:
That's a fair argument @BenjaSanchez. There would need to be a way to deal with the inherent conflicts though: `master` contains "new" models, `develop` contains (say) new pipeline code and also "new" models, merging `develop` into `master` would create conflicts. One easy way to do this would be to specify the `git merge` strategy (`git merge -s ours`).

Reply:
I like the idea of converting to an open `tsv` file.

Reply:
@mihai-sysbio that's a good point. How about then if any new pipeline code is implemented, we create it in a `feat/...` branch instead, we allow PRs that update ecModels to be created (to test if the new code works as intended), but we **don't merge** them to `feat/...`, to avoid conflicts. The new feature then gets merged to `master`, triggering then the actual update PRs. Would this work?

Reply:
It could work, but during development we would need to close the "broken" PRs, where the resulting models/PR body/whatever else is not correct. The PR target is specified in `config.ini`:

https://github.com/SysBioChalmers/ecModels/blob/2dbfb70ed89931be12654daec5cf2f9744cf9c98/config.ini#L3

Reply:
I might have misunderstood then how the current setup works, so all update PRs will be made to `master`, regardless of which branch triggers the PR? In the previous example, creating branch `feat/...` will trigger update PRs to `master` with the changes in `feat/...` + any ecModel changes after running the updated pipeline? Maybe we should schedule a call to land on a strategy that suits us best :)

Reply:
Yes @BenjaSanchez your conclusion is correct (based on the current implementation). I agree with you a call could be a quicker and probably more thorough way to discuss this, then we can update the discussion in this PR with whatever we have concluded.